spec/appboy/rest/schedule_messages_spec.rb

```diff
@@ -38,4 +60,19 @@ def expect_schedule_messages_http_call
       }
     })
   end
+
+  def expect_schedule_messages_update_http_call
+    expect(http).to receive(:post).with('/messages/schedule/update', {
+      app_group_id: app_group_id,
```

Comment:
Break these parameters into their own method and use that for expect calls in both create and update.